scripts/release/slack.ts

```diff
@@ -0,0 +1,35 @@
+import { WebClient } from '@slack/web-api'
+
+async function slack() {
+  if (!process.env.SLACK_TOKEN) {
+    throw new TypeError('SLACK_TOKEN not set')
+  }
+
+  const workflowLink = process.env.WORKFLOW_LINK
+  if (!workflowLink) {
+    throw new TypeError('WORKFLOW_LINK not set')
+  }
+
+  const message =
+    process.env.RELEASE_STATUS === 'true'
+      ? `Successfully published a new release!\n<https://github.com/vercel/next.js/releases|Releases Link>`
+      : `Failed to publish a new release triggered by "${process.env.WORKFLOW_ACTOR}". @nextjs-oncall\n<${workflowLink}|Workflow Link>`
```

Comment:
In combination with making the channel configurable for testing, you want the pinged person configureable as well. You wouldn't want to ping the on-call for testing.